common/thorhelper/roxiehelper.cpp

```diff
@@ -1771,6 +1780,8 @@ class HttpResponseHandler
             compression = respCompression;
         header.append("HTTP/1.0 200 OK\r\n");
         header.append("Content-Type: ").append(mlFmt == MarkupFmt_JSON ? "application/json" : "text/xml").append("\r\n");
+        if (httpKeepAlive)
+            header.append("Connection: Keep-alive\r\n");
```

Comment:
Keep-Alive

Reply:
Do we still want to use HTTP/1.0, or should we switch it to HTTP/1.1, and if httpKeepAlive is false, set "Connection: Close" ?

Reply:
I didn't want to chance any interop issues just for this issue.  We can evaluate changing to HTTP/1.1 separately.